config.go

```diff
@@ -125,6 +127,8 @@ func (c *agentConfig) parseCmdlineOption(option string) error {
 			agentLog.Debug("Param passed to NOT use vsock channel")
 			commCh = serialCh
 		}
+	case kmodulesFlag:
+		kmodules = strings.Split(split[valuePosition], kmodulesSeparator)
```

Comment:
What about adding the module list to `CreateSandboxRequest`?

Reply:
Also we should add module options for each module. Then grpc is a better place so that we have structured parameters. sth. like:
```go
type KernelModule struct{
        Name string
        options []string
}

type CreateSandboxRequest struct {
<snip...>
        KernelModules []KernelModule
<snip...>
}
```

Reply:
Talking about gRPC, how about updating `agentGRPC.GetGuestDetails()` so that it optionally returns the contents of `/proc/modules`? That could be useful for debug given that loading "module a" might auto-load "modules b-e".

Reply:
@jodh-intel @bergwolf  I agree, I like both ideas